src/lib/y2network/sysconfig/interfaces_reader.rb

```diff
@@ -108,7 +108,7 @@ def build_physical_interface(data)
         Y2Network::PhysicalInterface.new(data["dev_name"]).tap do |iface|
           iface.description = data["name"]
           type = data["type"] || Yast::NetworkInterfaces.GetTypeFromSysfs(iface.name)
-          iface.type = type.nil? ? :eth : type.to_sym
+          iface.type = type.nil? ? "eth" : type
```

Comment:
Shouldn't it be an `InterfaceType`?

Reply:
At least according to https://github.com/yast/yast-network/pull/888/files#diff-c8c2a4d640ee3258722656a5a52190f7R30

Reply:
you are right.